### ISSUE TITLE
BatchNorm checkerboard artifacts appearing on sampling with PYLD

### ISSUE THREAD

**Issue by lorenzo-querol** [sndnyang/JEMPP#2]
I have been trying to use Pytorch Lightning to reproduce your code from the public repository, where most of it is being transferred "as is". However, when I run the experiment on a different dataset (3x28x28 RGB images), with the same training configuration as CIFAR10, I get checkerboard artifacts that appear during sampling with your proposed PYLD.

![image](https://github.com/sndnyang/JEMPP/assets/10110005/8a8e448f-ad2f-4999-a42c-81cbeafff651)

Any reason why this is happening or might you know how I can solve this?

**Comment by sndnyang** [comment]
well, sorry for the late reply.  Did you solve it?
Did you reproduce the CIFAR10 successfully? But the new dataset failed, right?
Or my code works well on CIFAR10, but the Lightning framework failed to reproduce the results.
